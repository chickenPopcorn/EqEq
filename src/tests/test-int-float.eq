SomeCtx = {
  a = { 42; }
  b = { 4.2e-4; }
  c = { 0.42; }
  d = { .42; }
  e = { 0e1; }

}

SomeCtx:find a {
  print("%0.0f\n", a);
  printf("%0.5f\n", b);
  printf("%0.2f\n", c);
  printf("%0.2f\n", d);
  printf("%0.1f\n", e);
}
