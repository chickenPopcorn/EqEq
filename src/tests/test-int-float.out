42
0.00042
0.42
0.42
0.0
42000
-42
